dev-libs/mini-xml/mini-xml-2.12.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools
+
+MY_PN="mxml"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A small XML parsing library that you can use to read XML data files or strings"
+HOMEPAGE="https://github.com/michaelrsweet/mxml
+	https://www.msweet.org/mxml/"
+SRC_URI="https://github.com/michaelrsweet/${MY_PN}/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+LICENSE="Mini-XML"
+SLOT="0"
+IUSE="static-libs threads"
+
+BDEPEND="virtual/pkgconfig"
+
+S="${WORKDIR}/${MY_P}"
+
+src_prepare() {
+	# Respect users CFLAGS
+	sed -e 's/-Os -g//' -i configure.ac || die
+
+	# Don't run always tests
+	# Enable verbose compiling
+	sed -e '/ALLTARGETS/s/testmxml//g' -e '/.SILENT:/d' -i Makefile.in || die
+
+	# Build only static-libs, when requested by user, also build docs without static-libs in that case
+	if ! use static-libs; then
+		local mysedopts=(
+			-e '/^install:/s/install-libmxml.a//g'
+			-e '/^mxml.xml:/s/-static//g'
+			-e '/^mxml.epub:/s/-static//g'
+			-e '/^valgrind/s/-static//g'
+			-e '/.\/mxmldoc-static/s/-static//g'
+		)
+		sed ${mysedopts[@]} -i Makefile.in || die
+	fi
+
+	default
```

Comment:
You want to `default` before the `sed`s. This will ensures that patches made against vanilla upstream sources will apply cleanly.

Reply:
Done.